SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/ApplicationLifecycleHelperDesktop.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#if !NET5_0
+#if !WINDOWS10_0_17763_0
```

Comment:
Maybe it would be more readable if we define a compile constant with name `WINUI`.